src/Event/EventManager.php

```diff
@@ -103,14 +103,14 @@ public function on($eventKey = null, $options = [], $callable = null)
             return $this;
         }
         $argCount = func_num_args();
-        if ($argCount === 2) {
+        if ($eventKey && $argCount === 2) {
```

Comment:
`public function on($eventKey = null, `
if `$eventKey` can not be `null` in within this method, and it throw `InvalidArgumentException`
why it is `null`  by default ?

Reply:
Good question, I don't see why `$eventKey` is nullable in the interface. I'll remove it.